script/build.js

```diff
@@ -62,7 +62,7 @@ async function parseFile (file) {
 
   // derive props from the HTML
   const $ = cheerio.load(file.html || '')
-  file.title = $('h1').text()
+  file.title = $('h1').text() !== '' ? $('h1').text() : $('h2').text() 
```

Comment:
I think just `$('h1').text() || $('h2').text()` will work.

```
$ node
> a = '' || 'secondary'
'secondary'
> a
'secondary'
```

Reply:
TIL that's a thing...will do